pyproject.toml

```diff
@@ -14,6 +14,7 @@ dependencies = [
   "pandas",
   "tiktoken>=0.8.0",
   "tabulate",
+  "mlflow-skinny",
```

Comment:
Do we want to enforce a specific version that has tracing like 2.15 or something?

Reply:
good idea, will do min >2.19 bc we need some recent tracing fixes